<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoLXMPPDebuggerClass</class>
 <widget class="QMainWindow" name="LoLXMPPDebuggerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoLXMPPDebugger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="incomingLabel">
        <property name="text">
         <string>Incoming</string>
        </property>
        <property name="buddy">
         <cstring>incomingList</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="incomingList">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="incomingButtonClear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="incomingScrollToBottom">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Scroll to bottom</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="outgoingLabel">
        <property name="text">
         <string>Outgoing</string>
        </property>
        <property name="buddy">
         <cstring>outgoingList</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="outgoingList">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="outgoingButtonClear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="outgoingScrollToBottom">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Scroll to bottom</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionIncoming"/>
     <addaction name="actionOutgoing"/>
     <addaction name="actionBoth"/>
     <addaction name="separator"/>
     <addaction name="actionChoose_directory"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionIncoming">
   <property name="text">
    <string>Incoming</string>
   </property>
  </action>
  <action name="actionOutgoing">
   <property name="text">
    <string>Outgoing</string>
   </property>
  </action>
  <action name="actionBoth">
   <property name="text">
    <string>Both</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionChoose_directory">
   <property name="text">
    <string>Choose directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>incomingList</tabstop>
  <tabstop>incomingButtonClear</tabstop>
  <tabstop>incomingScrollToBottom</tabstop>
  <tabstop>outgoingList</tabstop>
  <tabstop>outgoingButtonClear</tabstop>
  <tabstop>outgoingScrollToBottom</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="LoLXMPPDebugger.qrc"/>
 </resources>
 <connections/>
</ui>
